src/components/CreateDrinkForm.js

```diff
@@ -0,0 +1,464 @@
+          </label>
+          <label htmlFor="measure3">
+            Measure 3:
+            <FormInput
+              type="text"
+              id="measure3"
+              placeholder="e.g. 3 oz"
+              {...register('measure3', {
+                minLength: {
+                  value: 2,
+                  message: 'This is too short',
+                },
+                maxLength: {
+                  value: 10,
+                  message: 'This is too long',
+                },
+              })}
+            />
+            <ErrorMessage>{errors.measure3?.message}</ErrorMessage>
+          </label>
+          <label htmlFor="ingredient4">
+            Ingredient 4:
+            <FormInput
+              type="text"
+              id="ingredient4"
+              placeholder="e.g. Orange juice"
+              {...register('ingredient4', {
+                minLength: {
+                  value: 2,
+                  message: 'This is too short',
+                },
+                maxLength: {
+                  value: 30,
+                  message: 'This is too long',
+                },
+              })}
+            />
+            <ErrorMessage>{errors.ingredient4?.message}</ErrorMessage>
+          </label>
+          <label htmlFor="measure4">
+            Measure 4:
+            <FormInput
+              type="text"
+              id="measure4"
+              placeholder="e.g. 3 oz"
+              {...register('measure4', {
+                minLength: {
+                  value: 2,
+                  message: 'This is too short',
+                },
+                maxLength: {
+                  value: 10,
+                  message: 'This is too long',
+                },
+              })}
+            />
+            <ErrorMessage>{errors.measure4?.message}</ErrorMessage>
+          </label>
+          <label htmlFor="ingredient5">
+            Ingredient 5:
+            <FormInput
+              type="text"
+              id="ingredient5"
+              placeholder="e.g. Orange juice"
+              {...register('ingredient5', {
+                minLength: {
+                  value: 2,
+                  message: 'This is too short',
+                },
+                maxLength: {
+                  value: 30,
+                  message: 'This is too long',
+                },
+              })}
+            />
+            <ErrorMessage>{errors.ingredient5?.message}</ErrorMessage>
+          </label>
+          <label htmlFor="measure5">
+            Measure 5:
+            <FormInput
+              type="text"
+              id="measure5"
+              placeholder="e.g. 3 oz"
+              {...register('measure5', {
+                minLength: {
+                  value: 2,
+                  message: 'This is too short',
+                },
+                maxLength: {
+                  value: 10,
+                  message: 'This is too long',
+                },
+              })}
+            />
```

Comment:
You could think about creating a component for the ingredient and measure part cause the code is more or less the same

Reply:
Yeah the whole form will be optimized later ;)